docs/integrate/datagrip/index.md

```diff
@@ -0,0 +1,108 @@
+(datagrip)=
+# DataGrip
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://blog.jetbrains.com/wp-content/uploads/2019/01/datagrip_icon.svg){w=120px}](https://www.jetbrains.com/datagrip/)
+```
+
+[DataGrip] is a cross-platform database IDE that is tailored to suit the
+specific needs of professional SQL developers.
+
+It is available as a standalone application and is also included into
+other JetBrains products like IntelliJ IDEA and PyCharm.
+
+Connecting DataGrip to CrateDB uses the [CrateDB JDBC Driver].
+
+```{div}
+:style: "clear: both"
+```
+
+
+## Install
+
+::::{grid} 2
+:::{grid-item}
+For connecting to CrateDB, install the [CrateDB JDBC Driver]
+using the "Custom JARs" option when adding a database driver.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/a8c1ada6-fd97-43f4-a1ba-91aba1520bdb){h=180px}
+![Image](https://github.com/user-attachments/assets/1f925848-fac3-4265-8bd3-96f91daf03c9){h=180px}
+:::
+:::{grid-item}
+[crate-jdbc-standalone] is the right choice here.
+For example, download and use the [crate-jdbc-standalone-2.7.0.jar] JAR file,
+and select the driver class `io.crate.client.jdbc.CrateDriver`.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/50ccb304-5aaf-4f0b-8ae7-55445f06930c){w=400px}
+:::
+::::
+
+
+## Connect
+
+::::{grid} 2
+
+:::{grid-item}
+Now, you can add a Data Source using the CrateDB database driver.
+Please specify database URL and credentials of your CrateDB cluster.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/147a3e8e-f1d7-413d-9e0c-1ced11333646){w=480px}
+:::
+:::{grid-item}
+When connecting to [CrateDB Self-Managed] on localhost,
+for evaluation purposes, use:
+```
+jdbc:crate://localhost:5432/
+```
+
+When connecting to [CrateDB Cloud], use:
+```
+jdbc:crate://<clustername>.cratedb.net:5432/
```

Comment:
This is not exactly `<clustername>.cratedb.net` as region and cloud vendor are also part of the URL, but I guess it is still ok?

Reply:
Yeah, we are using both variants in documentation. The other variant, because it includes cloud vendor and region, has many subvariants, so I guess the idea was to canonicalize, also in order to have a more shorter representation. Maybe I picked this variant up from cloud docs somewhere?

If you have any suggestions how to represent a canonical version for a CrateDB Cloud URL better, please let me know.

/cc @proddata, @surister

Reply:
Do we even have to discriminate between local and cloud cluster here?

```
jdbc:crate://<host>:5432/
```

Reply:
Not necessarily. Updated.